### ISSUE TITLE
Finetune based on user feedback (iteration 1)

### ISSUE THREAD

**Issue by webbertakken** [webbertakken/reviewer#25]
**Context**

Fine-tune comments placed by this app, based on feedback from end-users.

Of course there's always something to improve, but the expectation is that people will find use cases that don't exactly play well with our current implementation.

**Suggested solution**

Listen to the community. Solve problems based on feedback.

**Considered alternatives**

N/A

**Additional details**

Use the bug tag: https://github.com/webbertakken/reviewer/labels/bug
